pkgs/top-level/aliases.nix

```diff
@@ -1626,6 +1626,7 @@ mapAliases ({
   xfceUnstable = xfce4-14; # Added 2019-09-17
   xineLib = xine-lib; # Added 2021-04-27
   xineUI = xine-ui; # Added 2021-04-27
+  xlibsWrapper = throw "'xlibsWrapper' has been replaced by it's constituents"; # Converted to throw 2022-12-27
```

Comment:
It should be `its`, not `it's`. Thank you for your working on deprecating this old meta package :-)

Reply:
Thank you! Updated `it's` to `its` everywhere.